pkg/config/config.go

```diff
@@ -133,11 +136,12 @@ func GetConfig() (Config, error) {
 	}
 
 	conf := &Config{
-		MaxTestRetries: 2,
-		MaxRetries:     3,
-		LogLevel:       logrus.InfoLevel.String(),
-		Project:        "runiac",
-		TargetAll:      true,
+		MaxTestRetries:     2,
+		MaxRetries:         3,
+		TimeBetweenRetries: 0,
```

Comment:
@samar-upadhyay the previous timebetweenretries was 5 - correct?  is that still the default here?